UnchainedLauncherCore/src/API/A2S.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.Buffers.Binary;
+
+namespace UnchainedLauncher.Core.API
+{
+    public record A2S_INFO (
+        byte protocolVersion, 
+        String name,
+        String map,
+        String folder,
+        String game, 
+        UInt16 gameID, 
+        byte players,
+        byte maxPlayers,
+        byte bots, 
+        ServerType serverType, 
+        Environment environment, 
+        bool isPublic, 
+        bool vac
+        );
+
+    public enum ServerType : byte
+    {
+        DEDICATED = (byte)'D',
+        NONDEDICATED = (byte)'L',
+        PROXY = (byte)'P'
+    }
+
+    public enum Environment : byte
+    {
+        LINUX = (byte)'L',
+        WINDOWS = (byte)'W',
+        MAC = (byte)'M'
+    }
+
+    public static class A2S
+    {
+        const int timeOutMillis = 1000;
```

Comment:
This should be PascalCase as well. Basically any identifier that's not a parameter should be PascalCase... Though I've been using camel case for parameters and local values/variables within functions. 

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions